app/src/helpers/loggingHelper.ts

```diff
@@ -0,0 +1,76 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+import * as loglevel from 'loglevel';
+
+import { safeGetEnv } from './playwrightHelpers';
+
+const USE_LOG_FILE = safeGetEnv('USE_LOG_FILE') === '1';
+const LOG_FILE_DIR = safeGetEnv('LOG_FILE_DIR') ?? process.cwd();
+const LOG_FILENAME = path.join(LOG_FILE_DIR, `${new Date().getTime()}.log`);
+
+const logLevelNames = ['TRACE', 'DEBUG', 'INFO ', 'WARN ', 'ERROR'];
+
+function _logger(
```

Comment:
Johann-Sebastian Doc with a rationale for this new wrapper to help future devs, pls!